authors/apps/notifications/tests/test_create_and_fetch.py

```diff
@@ -0,0 +1,94 @@
+"""Module tests creation and fecthing of notifications"""
+from rest_framework.test import APIClient
+from django.urls import reverse
+from django.test import TestCase
+from rest_framework import status
+from ..models import Notification
+from ..models import Article
+
+
+class TestSetup(TestCase):
+    """Act as base class for all other test classes"""
+    def setUp(self):
+        """Initialize all testing data"""
+        self.client = APIClient()
+        #register a user to login with
```

Comment:
Work on pep 8 especially where we have comments and docstrings.

Reply:
Working on it.